CONTRIBUTING.md

```diff
@@ -281,11 +281,21 @@ clean the docker disk space periodically.
 If you are on MacOS:
 
 * Run `brew install gnu-getopt coreutils` (if you use brew, or use equivalent command for ports)
-* Then (with brew) link the gnu-getopt to become default as suggested by brew by typing.
+* Then (with brew) link the gnu-getopt to become default as suggested by brew by typing:
 ```bash
-echo 'export PATH=\"/usr/local/opt/gnu-getopt/bin:\$PATH\"' >> ~/.bash_profile"
-. ~/.bash_profile"
+echo 'export PATH="/usr/local/opt/gnu-getopt/bin:$PATH"' >> ~/.bash_profile
+. ~/.bash_profile
 ```
+
+if you use bash or
```

Comment:
The sentence structure, which first shows the command and then the condition, is difficult to understand. Can you write a sentence so that the condition first and then the command?
Example: 
> If you are using bash,  then you should execute the command:
> If you are using zsh, then you should execute the command:
Sentences are correct, but in the presented construction they will be easier to understand.

Reply:
Good point. fixed